src/main/resources/db/changelog/v.0.2/db.changelog-master.yaml

```diff
@@ -8,5 +8,11 @@ databaseChangeLog:
  - include:
       file: db.changelog-interview-dates.yaml
       relativeToChangelogFile: true
+ - include:
```

Comment:
why do you put the files in v.0.2?
current version of liquibase changes is v.0.3

Reply:
Done

Reply:
Please, check again. These files are in not correct directory and included to incorrect changelog-master.yaml

Reply:
@Fenydude ^^^

Reply:
AAAAAAA I can't understand what should i do. I did changeSet for internship and form on Friday and Saturday, and therefore i store that file in v.0.2

Reply:
In current situation we decided to follow version despite of absence of production release. And if all start working from the beginning week with v0.3 but PR is still open, scripts should be moved to recent version folder (v0.3).